### DISCUSSION TITLE
Prev Next button in tabbed

### DISCUSSION THREAD

**Discussion by SPyofgame** [squidfunk/mkdocs-material#5260] [Q&A] [answered]
What is this button for

![image](https://user-images.githubusercontent.com/62207095/227733075-1d03345e-81df-4f23-b24b-0a61c04c1376.png)

**Comment by facelessuser** [comment, marked as answer]
For when you have so many tabs that they overflow:

![tabs](https://user-images.githubusercontent.com/1055125/227782313-d2f65de0-23bd-41fa-b7af-58825fb8b195.gif)

**Reply by SPyofgame** [reply]
Oh that was unexpected, I thought that was for mobile only
